users/mnil/mnil.c

```diff
@@ -0,0 +1,133 @@
+#include QMK_KEYBOARD_H
+#include "config.h"
```

Comment:
Not needed.  `config.h` gets included in every file as part of the build process. No need to explicitly include it. 
```suggestion
```